app/helpers/brexit_checker_helper.rb

```diff
@@ -1,6 +1,10 @@
 require "addressable/uri"
 
 module BrexitCheckerHelper
+  def encoded_results_url
```

Comment:
no test for this, but it's a fairly simple alias.
I think I'd need to stub the request to unit test that here?